relayer/crates/starknet-chain-components/src/impls/starknet_to_cosmos/connection_message.rs

```diff
@@ -214,7 +246,8 @@ where
         let message = CosmosConnectionOpenConfirmMessage {
             connection_id: connection_id.to_string(),
             update_height,
-            proof_ack: counterparty_payload.proof_ack.proof_bytes,
+            // FIXME(rano): generate the signature
+            proof_ack: dbg!(counterparty_payload.proof_ack.proof_bytes),
```

Comment:
Currently, the signature verification fails here. But I don't know how to create the signature at this point without access to `ConnectionEnd` and `ConnectionId` at starknet.

Reply:
I can query the stored counteryparty connection end (starknet) at cosmos, cosmos's own connection id to recreate the ConnectionEnd.